pkg/catalog/errors.go

```diff
@@ -13,12 +11,12 @@ var (
 	ErrRequiredValue            = fmt.Errorf("required value: %w", ErrInvalid)
 	ErrPathRequiredValue        = fmt.Errorf("missing path: %w", ErrRequiredValue)
 	ErrInvalidValue             = fmt.Errorf("invalid value: %w", ErrInvalid)
-	ErrNotFound                 = db.ErrNotFound
+	ErrNotFound                 = errors.New("not found")
 	ErrInvalidMetadataSrcFormat = errors.New("invalid metadata src format")
 	ErrExpired                  = errors.New("expired from storage")
 	ErrFeatureNotSupported      = errors.New("feature not supported")
 	ErrBranchNotFound           = fmt.Errorf("branch %w", ErrNotFound)
-	ErrRepositoryNotFound       = fmt.Errorf("repository %w", ErrNotFound)
+	ErrRepositoryNotFound       = fmt.Errorf("repository: %w", ErrNotFound)
```

Comment:
if we update repository error to use ':' - I think we need to align branch too.

Reply:
Reverted this change. It appears in too many places to start refactoring it as part of this PR.